### PR TITLE
add forcedExtension to ITextureCreationOptions

### PR DESCRIPTION
**Pull request by noname0310**
The forcedExtension option seems to be missing from ITextureCreationOptions.

Currently I am using the ITextureCreationOptions to create the texture:
```typescript
const tex = new Texture(
    "url...",
    scene,
    {
        noMipmap: true,
        invertY: false,
        samplingMode: Texture.NEAREST_SAMPLINGMODE,
        onLoad: (): void => {
            // do something
        }
    }
);
```

But in order to use forcedExtension here, the code should look like this:
```typescript
const tex = new Texture(
    "url...",
    scene,
    {
        noMipmap: true,
        invertY: false,
        samplingMode: Texture.NEAREST_SAMPLINGMODE,
        onLoad: (): void => {
            // do something
        }
    },
    undefined, // invertY
    undefined, // samplingMode
    undefined, // onLoad
    undefined, // onError
    undefined, // buffer
    undefined, // deleteBuffer
    undefined, // format
    undefined, // mimeType
    undefined, // loaderOptions
    undefined, // creationFlags
    ".bmp" // forcedExtension
);
```

I guess this is not the intended design. That's why this change is proposed.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15872/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15872/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15872/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15872/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15872/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15872/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15872/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15872/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15872/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15872/merge/testResults/webgl2playwright/index.html

**Review by RaananW — APPROVED** [review]

File: packages/dev/core/src/Materials/Textures/texture.ts [hunk 1]
```diff
@@ -431,6 +434,7 @@ export class Texture extends BaseTexture {
             useSRGBBuffer = noMipmapOrOptions.useSRGBBuffer ?? false;
             internalTexture = noMipmapOrOptions.internalTexture ?? null;
             gammaSpace = noMipmapOrOptions.gammaSpace ?? gammaSpace;
+            forcedExtension = noMipmapOrOptions.forcedExtension;
```
Comment (sebavan) [inline packages/dev/core/src/Materials/Textures/texture.ts:437]:
```suggestion
            forcedExtension = noMipmapOrOptions.forcedExtension ?? forcedExtension;
```

**Review by sebavan — APPROVED** [review]

**Pull request merged**